es-vue-base/tests/EsBadge.spec.js

```diff
@@ -0,0 +1,70 @@
+import { mount } from '@vue/test-utils';
+import EsBadge from '@/src/lib-components/EsBadge.vue';
+import jestVue from '@/tests/jest.vue.config';
+
+describe('EsBadge', () => {
+    // Basic test; should exist for most components
+    const variantOptions = ['primary', 'secondary', 'danger', 'warning', 'info', 'light', 'dark', 'link'];
+    const outlineOptions = [true, false];
+    const slots = {
+        default: 'Main Content',
+    };
+    const paramaters = [];
+    variantOptions.forEach((variant) => {
+        outlineOptions.forEach((outline) => {
+            paramaters.push({
+                variant,
+                outline,
+            });
+        });
+    });
+
+    test.each(paramaters)(
```

Comment:
I can across `test.concurrent.each` which seems to speed things up.

Reply:
Looks like it was some issues with `async` events

`thrown: "Axe is already running. Use `await axe.run()` to wait for the previous run to finish before starting a new run."`